ebos/eclbasevanguard.hh

```diff
@@ -193,6 +195,8 @@ public:
      */
     EclBaseVanguard(Simulator& simulator)
         : ParentType(simulator)
+        , summaryState_()
```

Comment:
nit: if you don't need to explicitly call the default constructor of an attribute.